FAQ.md

```diff
@@ -0,0 +1,50 @@
+## Why is my drive not bootable?
+
+Etcher copies images to drives byte by byte, without doing any transformation to the final device, which means images that require special treatment to be made bootable, like Windows images, will not work out of the box. In these cases, the general advice is to use software specific to those kind of images, usually available from the image publishers themselves. You can find more information [here](https://github.com/balena-io/etcher/blob/master/docs/USER-DOCUMENTATION.md#why-is-my-drive-not-bootable).
+
+## How can I configure persistent storage?
+
+Some programs, usually oriented at making GNU/Linux live USB drives, include an option to set persistent storage. This is currently not supported by Etcher, so if you require this functionality, we advise to fallback to [UNetbootin](https://unetbootin.github.io/).
+
+## How can I deactivate desktop shortcut prompt on GNU/Linux?
+
+This is a feature provided by [AppImages](https://github.com/balena-io/etcher/blob/master/docs/appimage), where the applications prompts the user to automatically register a desktop shortcut to easily access the application. Follow the steps you can find [here](https://github.com/balena-io/etcher/blob/master/docs/USER-DOCUMENTATION.md#deactivate-desktop-shortcut-prompt-on-gnulinux).
+
+## How do I flash Ubuntu ISOs
+
+Ubuntu images (and potentially some other related GNU/Linux distributions) have a peculiar format that allows the image to boot without any further modification from both CDs and USB drives.
+A consequence of this enhancement is that some programs, like parted get confused about the drive's format and partition table, printing warnings such as: \n/dev/xxx contains GPT signatures, indicating that it has a GPT table. However, it does not have a valid fake msdos partition table, as it should. Perhaps it was corrupted -- possibly by a program that doesn't understand GPT partition tables. Or perhaps you deleted the GPT table, and are now using an msdos partition table. Is this a GPT partition table? Both the primary and backup GPT tables are corrupt. Try making a fresh table, and using Parted's rescue feature to recover partitions.
```

Comment:
IMHO it's not clear which is part of the "FAQ text" and which are the "possible warning messages".

Reply:
Please add double quotes or format it as a quote (with >)